Analysis/Tasks/PWGCF/FemtoDream/femtoDreamPairTask.cxx

```diff
@@ -42,8 +43,10 @@ struct femtoDreamPairTask {
 
   void process(aod::FemtoDreamCollision const& col, aod::FemtoDreamParticles const& parts)
   {
+    float multCol = col.multV0M();
     for (auto& [p1, p2] : combinations(parts, parts)) {
       sameEventCont.setPair(p1, p2);
+      sameEventCont.SetEventPairHistograms(multCol, p1, p2);
```

Comment:
Because the k* for each pair can vary and since in SetEventPairHistograms I have to fill the histogram Multiplicity vs k* it seemed the right thing to do.